internal/lpastore/update.go

```diff
@@ -36,6 +36,14 @@ func (c *Client) sendUpdate(ctx context.Context, lpaUID string, actorUID actorui
 	return c.do(ctx, actorUID, req, nil)
 }
 
+func (c *Client) SendRegister(ctx context.Context, lpaUID string) error {
```

Comment:
I don't really like this, but the alternative was to make `SendUpdate` or `Do` public, which I think would be worse

Reply:
Fair compromise 👍